From: Hideki Yamane <henrich@debian.org>
Date: Sat, 24 Oct 2015 17:15:45 +0900
Subject: adjust FDKScripts path for debian package

---
 FDK/Tools/linux/setFDKPaths | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: afdko/FDK/Tools/linux/setFDKPaths
===================================================================
--- afdko.orig/FDK/Tools/linux/setFDKPaths
+++ afdko/FDK/Tools/linux/setFDKPaths
@@ -7,7 +7,7 @@ AFDKO_EXE_PATH="${cmdDirPath}"
 # uncomment this to point to the system python
 AFDKO_Python="/usr/bin/env python"
 
-AFDKO_Scripts="${AFDKO_EXE_PATH}"/../SharedData/FDKScripts
+AFDKO_Scripts=/usr/share/afdko/FDKScripts
 
 export AFDKO_EXE_PATH
 export AFDKO_Python
